packages/manager/modules/manager-components/src/edit-name/template.html

```diff
@@ -0,0 +1,29 @@
+<form
+    id="serverEditName"
+    name="serverEditName"
```

Comment:
Good practice to prefix it with "Form":

```suggestion
    name="serverEditNameForm"
```